bears/coffee_script/CoffeeLintBear.py

```diff
@@ -239,6 +241,10 @@ class Foo
         :param allow_empty_functions:
             Allows declaring empty functions.
         """
+        # max_line_length set to maximum safe int in CoffeScript (JS).
+        if(max_line_length == 0):
```

Comment:
no need of brackets, and better change it to `if not max_line_length`. rather than allotting direcctly better make it 
`var max_safe_ int = 2** 53-1`
and then allot it

Reply:
No, `not max_line_length` is not explicit. `max_line_length == 0` is better. I agree with the point about brackets though.